### PR TITLE
Add Text Input Android documentation

### PR DESCRIPTION
**Pull request by byamba3**
Add android documentation for Text Input.

### CODE REVIEW

**Comment by danoc** [comment]
Looks good to me. @lasya-tack, could you please take a look as wel?

File: www/src/pages/components/text-input/android/index.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { ComponentHeader } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+### Components
+
+There are four types of Text Input components:
+
+-   `ThumbprintTextInput` : A single line `ThumbprintEditText` with no drawables. While you can use `ThumbprintTextInputWithDrawables` without defining any drawables in place of this View, it is recommended to use this View if you do not plan on defining drawables. This is because `ThumbprintTextInputWithDrawables` has tap event listeners which will do unncessary work if no drawables are defined.
+-   `ThumbprintTextInputWithDrawables` : A single line `ThumbprintEditText` with drawables. Contains logic for capturing clicks on drawables (both start and end). Can expose drawable tap events through `.uiEvents()`.
```
Comment (lasya-tack) [inline www/src/pages/components/text-input/android/index.mdx:16]: 1. > Can expose drawable tap events through `.uiEvents()`.

I don't think this is true? Looking at the implementation, we don't actually expose any UIEvents. Instead, you can use `drawableStartListener` and `drawableEndListener` to listen to drawable taps.

2. Perhaps add a note saying only start & end drawables are recommended?

File: www/src/pages/components/text-input/android/index.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+-   `ThumbprintTextArea` : A multi-line ThumbprintEditText. For iOS and web parity, this is referred to as a TextArea.
```
Comment (lasya-tack) [inline www/src/pages/components/text-input/android/index.mdx:17]: Should this go under [documentation for `TextArea`](https://thumbprint.design/components/text-area/react/) component instead? 🤔

File: www/src/pages/components/text-input/android/index.mdx (hunk 1 above)
Comment (byamba3) [reply www/src/pages/components/text-input/android/index.mdx]: Good point. I just copied the description from the `ThumbprintTextInputWithDrawables.kt` file itself. I guess the description was always misleading. I've updated it to specify the listeners.

File: www/src/pages/components/text-input/android/index.mdx (hunk 2 above)
Comment (byamba3) [reply www/src/pages/components/text-input/android/index.mdx]: Done.

File: www/src/pages/components/text-area/android/index.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+<ComponentHeader data={props.data} />
+
+`ThumbprintTextArea` is a multi-line ThumbprintEditText. For iOS and web parity, this is referred to as a Text Area. Since `ThumbprintTextArea` inherits from `ThumbprintEditTextBase`, it shares common properties with `ThumbprintTextInput`.
+
+### States
+
+The following states are available for inputs: Default, selected, filled, disabled, and error.
+
+![Text Area states](/img/native-text-input-states.png 'text-input states')
```
Comment (lasya-tack) [inline www/src/pages/components/text-area/android/index.mdx:17]: This image shows the single line text-input component. Do you need to update it for TextArea?

**Review by lasya-tack — DISMISSED** [review]
Fix image & :shipit:

File: www/src/pages/components/text-area/android/index.mdx (hunk 3 above)
Comment (byamba3) [reply www/src/pages/components/text-area/android/index.mdx]: Ahh I was looking at the wrong part of the Figma file. Done and added.

**Review by danoc — APPROVED** [review]
Great, thanks Bryan! I'm merging this and it'll be live in ~8 mins.

**Pull request merged**